akka-typed/src/main/scala/akka/typed/EventStream.scala

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com/>
+ */
+package akka.typed
+
+import akka.{ event ⇒ e }
+import akka.event.Logging.{ LogEvent, LogLevel, StdOutLogger }
+import akka.testkit.{ EventFilter, TestEvent ⇒ TE }
```

Comment:
my two cents before forgetting them, I do not think it's worth to depend on testkit.
If needed ```EventFilter``` and ```TestEvent``` should be moved under akka-actor.